### PR TITLE
Add one note for commit style

### PR DESCRIPTION
**Pull request by akhatmullin**

### CODE REVIEW

**Comment by akhatmullin** [comment]
Add one note for commit style about "how to fix a mistake"

**Comment by zummenix** [comment]
Maybe it's better to place it in `commit-style.md`. What do you think?

**Comment by akhatmullin** [comment]
You're right. I will replace it.

**Comment by zummenix** [comment]
Thanks!

File: commit-style.md [hunk 1]
```diff
@@ -8,9 +8,11 @@
 5. Capitalize the subject line
 6. Do not end the subject line with a period
 7. Use the body for explain what and why vs. how.
+8. If you have made a mistake in your commit message, you can fix it using the commad git commit --amend
```
Comment (AgapovOne) [inline commit-style.md:11]: Also, you can use "Ammend" button in SourceTree git client.
And a typo here - commaNd

Comment (akhatmullin) [reply commit-style.md:11]: Thanks!
I really rarely use SourceTree and haven't seen this button.

**Pull request merged**